Adjusts issue extraction to the updated network library

fehnkera opened the pull request:
As the network library is adjusted to the new issue data format, now also the default 'print_to_disk' methods in 'issue_extraction.py' and 'jira_issue_extraction.py' use the new issue data format. 
Also some minor changes in the new issue data format were necessary, as lists are now saved in json representation.

Signed-off-by: Anselm Fehnker <fehnker@fim.uni-passau.de>

Commit:
Change list output format to json standard

The lists in the the 'bugs-jira.list' file are now printet out in json
standard, i. E. "[""text""]" instead of "['text']".

Signed-off-by: Anselm Fehnker <fehnker@fim.uni-passau.de>

Commit:
Change default issue format to the new format

The methods 'print_to_disk' in 'issue_processing.py' and in
'jira_issue_processing.py' now also use the new issue data fromat.

Signed-off-by: Anselm Fehnker <fehnker@fim.uni-passau.de>

clhunsen reviewed (approved):
Thanks for the changes and all the data consolidation, @fehnkera!

Just one comment from my side: Please adjust the copyright headers.

bockthom commented on issue_processing/jira_issue_processing.py:
```diff
@@ -566,20 +564,74 @@ def print_to_disk(issues, results_folder):
+            json.dumps(["unresolved"])  ## default resolution when created
+        ))
+
+        lines.append((
```
Just one comment here: Why do you call `lines.append` twice with almost the same parameters?
At first glance, I was confused and did not see that there is a little difference regarding `"created"` and `"commented"`. After talking to @clhunsen, I now  understand what's going on here. However, it would be helpful to have a comment in the code that explains why this event is added twice and what the difference is. So, @fehnkera, could you please add some documentation for that? Thanks!

bockthom reviewed (commented):
Thanks for those nice adjustments, @fehnkera. Looks good!

I only have identified one issue regarding code comprehensibility, please see that below.

clhunsen commented:
I am currently working on some follow-up work in the network library and I noticed some inconsistencies regarding column types. The best way to correct this is to ensure that the column `event.info.2` is always dumped with a wrapping `json.dumps` call – regardless of where the information for this column comes from. 

@fehnkera, can you please add the corresponding calls in the extraction? (I will make the changes in the network library then.)

Commit:
Dump complete 'event_info_2' column to json

Applies review from @clhunsen and @bockthom. This causes that the
complete 'event_inof_2' column is dumped to json, the copyright headers
are updated and some documentation is added.

Signed-off-by: Anselm Fehnker <fehnker@fim.uni-passau.de>

Commit:
Change name of output file for GitHub issues

The name of the 'print_to_disk' output file in 'issue_prcessing.py' is
changed from 'issues.list' to 'issues-github.list'. The naming of the
JIRA and GitHub issue output files is now consistent.

Signed-off-by: Anselm Fehnker <fehnker@fim.uni-passau.de>

clhunsen commented:
Thanks for your valuable changes, @fehnkera! I will merge now.
